service-front/module/Application/src/Controller/DonorFlowController.php

```diff
@@ -55,7 +55,11 @@ public function howWillDonorConfirmAction(): ViewModel|Response
                     $view->setVariables($formProcessorResponseDto->getVariables());
                 } else {
                     $this->opgApiService->updateIdMethod($uuid, $formData['id_method']);
-                    return $this->redirect()->toRoute("root/donor_details_match_check", ['uuid' => $uuid]);
+                    if ($formData['id_method'] == 'po') {
```

Comment:
It would be good to move these values into an enum (like `IdMethod` in the API) rather than using raw strings.

For example, this isn't in [the list of identity_methods](https://github.com/ministryofjustice/opg-paper-identity/blob/b9394ef9306bcdf91fee221dc8af99fa07b090ae/service-front/module/Application/config/module.config.php#L470-L474) so it's hard to follow where it comes from.